samples/cellular/nrf_cloud_mqtt_multi_service/overlay_nrf7002ek_wifi_no_lte.conf

```diff
@@ -0,0 +1,128 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Note that this overlay is not compatible with nrf9160! Use something such as the nrf5340 as the
+# host microcontroller.
+
+## Disable Modem/LTE/nrf91-specific features
+CONFIG_NRF_MODEM_LIB=n
+CONFIG_MODEM_INFO=n
+CONFIG_MODEM_INFO_ADD_NETWORK=n
+CONFIG_NRF_MODEM_LIB=n
+CONFIG_AT_HOST_LIBRARY=n
+CONFIG_LTE_LINK_CONTROL=n
+
+## Disable offloaded sockets (we are going to switch to the native networking stack)
+CONFIG_NET_SOCKETS_OFFLOAD=n
+
+## Disable cloud features reliant on or specific to nrf91 modem features
+CONFIG_NRF_CLOUD_AGPS=n
+CONFIG_NRF_CLOUD_PGPS=n
+CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT=n
+CONFIG_LOCATION_METHOD_GNSS=n
+CONFIG_LOCATION_METHOD_CELLULAR=n
+
+## Disable FOTA to conserve memory
+CONFIG_NRF_CLOUD_FOTA=n
+CONFIG_BOOTLOADER_MCUBOOT=n
+CONFIG_IMG_MANAGER=n
+CONFIG_MCUBOOT_IMG_MANAGER=n
+
+## Disable LTE-event-driven date_time updates (the date_time library will still periodically
```

Comment:
Do you mean it will still update using NTP?

Reply:
I specifically mean it will still periodically update itself without prompting. Yes, NTP will be used, but that's not my focus with that statement